src/systems/tunnel-effect.js

```diff
@@ -0,0 +1,164 @@
+import "../utils/postprocessing/EffectComposer";
+import "../utils/postprocessing/RenderPass";
+import "../utils/postprocessing/ShaderPass";
+import "../utils/postprocessing/MaskPass";
+import "../utils/shaders/CopyShader";
+import "../utils/shaders/VignetteShader";
+import qsTruthy from "../utils/qs_truthy";
+
+const isDisabled = qsTruthy("disableTunnel");
```

Comment:
On by default, then disabled with a query parameter will get you the most feedback. Add a note in README.md under the ##Query Params header.